examples/helia-101/101-basics.js

```diff
@@ -1,39 +1,61 @@
 /* eslint-disable no-console */
+// @ts-check
 
+import * as nodefs from 'fs'
+import { devNull } from 'node:os'
+import { pipeline } from 'stream/promises'
 import { createHeliaHTTP } from '@helia/http'
-import { unixfs } from '@helia/unixfs'
+import { unixfs, urlSource } from '@helia/unixfs'
 
-// create a Helia node
+// `@helia/http` is an light http-only version Helia with the same API,
+// which is useful for simple use cases, where you don't need p2p networking to provide data to other nodes.
+// Since this example is focused on UnixFS without p2p networking, we can use the `@helia/http` package.
 const helia = await createHeliaHTTP()
 
-// create a filesystem on top of Helia, in this case it's UnixFS
+// UnixFS allows you to encode files and directories such that they are addressed by CIDs and can be retrieved by other nodes on the network
 const fs = unixfs(helia)
 
-// add a file and wrap in a directory
-const readmeCid = await fs.addFile({
-  path: './README.md'
-}, {
-  wrapWithDirectory: true
-})
-
-console.log('Added README.md file:', readmeCid.toString())
-
-// we will use this TextEncoder to turn strings into Uint8Arrays
+// we will use this TextEncoder to turn strings into Uint8Arrays which we can add to the node
 const encoder = new TextEncoder()
 
-// add the bytes to your node and receive a unique content identifier
+// addBytes takes raw bytes and returns a raw block CID for the content
+// (larger (over 1 MiB) binary arrays are chunked and return a dag-pb block CID instead)
+// The `bytes` value we have passed to `unixfs` has now been turned into a UnixFS DAG and stored in the helia node.
 const cid = await fs.addBytes(encoder.encode('Hello World 101'), {
   onProgress: (evt) => {
     console.info('add event', evt.type, evt.detail)
   }
 })
-
 console.log('Added file:', cid.toString())
 
+// Create an empty directory
+const directoryCid = await fs.addDirectory({
+  path: 'my-dir'
+})
```

Comment:
With the `path` opt, we're actually creating a directory that contains a directory called `my-dir` (in order to preserve the filename).

If we just want an empty directory to add stuff too, do:

```suggestion
const directoryCid = await fs.addDirectory()
```